R/post-favorite.R

```diff
@@ -14,6 +14,9 @@
 #' }
 #' @family post
 #' @export
+#' @references 
+#' Create: <https://developer.twitter.com/en/docs/twitter-api/v1/tweets/post-and-engage/api-reference/post-favorites-create>
```

Comment:
You might want to make this a bulleted list.

Reply:
Well, I hope that once we agree on the naming convention we will split some of these functions where multiple endpoints are used from the same function, so I'll wait till then to add these cosmetic touches.